### ISSUE TITLE
Sync with Github profiles

### ISSUE THREAD

**Issue by kousu** [neuropoly/neuropoly-docs#25]
> re: pictures of lab members: how about fetching (if it exists) the picture from [github](https://github.com/orgs/neuropoly/people)? that would ensure most pictures are up there (without "pushing" lab members) and make it automated (for new members)

_Originally posted by @jcohenadad in https://github.com/neuropoly/neuropoly-docs/issues/21#issuecomment-959192573_

Given that we have our org-chart recorded on https://github.com/orgs/neuropoly/teams

...

it should be possible to generate most of `/team`. To make it easier to manage, we can use [`include`](https://myst-parser.readthedocs.io/en/latest/sphinx/use.html#include-a-file-from-outside-the-docs-folder-like-readme-md), e.g.:

    ## Faculty

    ```{include} _faculty.md
    ```

    ## PhD Students

    ```{include} _phd-students.md
    ```

    ...

where the script is responsible for generating `team/_*.md`.

It's really easy to make this go one-way. Where this gets tricky is with preserving people's custom profile links, alternate social media, email addresses. Not impossible, but it means actually parsing the pre-existing markdown and that's likely to be buggy.

We *could* tell people that *everything* will be pulled from github, so e.g. if they want an email address listed they need to also make it public *on github*, and if they want to use a personal site they need to list it publically *on github*.

**Comment by jcohenadad** [comment]
> We could tell people that everything will be pulled from github, so e.g. if they want an email address listed they need to also make it public on github, and if they want to use a personal site they need to list it publically on github.

yes, as i was reading your post, i though exactly about that option. That would be neat!
